lib/bugzilla_service.rb

```diff
@@ -13,26 +13,126 @@ class BugzillaService
     resolved
   )
 
+  # Helper class for specific bugs
+  class Bug
+    attr_reader   :service
+    attr_accessor :id, :status
+
+    def initialize(service, id, status)
+      @service, @id, @status = service, id, status
+    end
+
+    def comments
+      response = service.get("/rest/bug/#{id}/comment") do |req|
+        req.params["include_fields"] = "text"
+      end
+
+      if response.status == 200
+        JSON.parse(response.body)["bugs"][id.to_s]["comments"]
+            .map { |comment| comment["text"] }
+      else
+        []
+      end
+    end
+
+    # Adds a comment.  Returns true if status == 200
+    def add_comment(text)
+      payload  = {"comment" => text}.to_json
+      response = service.post("/rest/bug/#{id}/comment", payload)
+
+      response.status == 200
+    end
+
+    # Basicially API compatible with `active_bugzilla` way of updating the bug
+    # status.
```

Comment:
Oh I see, so with the REST API it *can't* be done in a single call?

Reply:
Answered above:  https://github.com/ManageIQ/miq_bot/pull/434#discussion_r256509574